insights/parsers/qemu_xml.py

```diff
@@ -156,3 +164,145 @@ def parse_dom(self):
+              <topology sockets='16' cores='2' threads='1'/>
+              <numa>
+                <cell id='0' cpus='0-1' memory='2097152' unit='KiB'/>
+              </numa>
+            </cpu>
+            <clock offset='variable' adjustment='1' basis='utc'>
+              <timer name='rtc' tickpolicy='catchup'/>
+              <timer name='pit' tickpolicy='delay'/>
+              <timer name='hpet' present='no'/>
+            </clock>
+            <on_poweroff>destroy</on_poweroff>
+            <on_reboot>restart</on_reboot>
+            <on_crash>destroy</on_crash>
+            <devices>
+              <emulator>/usr/libexec/qemu-kvm</emulator>
+              <disk type='file' device='cdrom'>
+                <driver name='qemu' type='raw'/>
+                <source startupPolicy='optional'/>
+                <backingStore/>
+                <target dev='hdc' bus='ide'/>
+                <readonly/>
+                <alias name='ide0-1-0'/>
+                <address type='drive' controller='0' bus='1' target='0' unit='0'/>
+              </disk>
+              <controller type='usb' index='0' model='piix3-uhci'>
+                <alias name='usb'/>
+                <address type='pci' domain='0x0000' bus='0x00' slot='0x01' function='0x2'/>
+              </controller>
+              <interface type='bridge'>
+                <mac address='00:1a:4a:16:03:72'/>
+                <source bridge='vlan2593'/>
+                <target dev='vnet20'/>
+                <model type='virtio'/>
+                <filterref filter='vdsm-no-mac-spoofing'/>
+                <link state='up'/>
+                <boot order='2'/>
+                <alias name='net0'/>
+                <address type='pci' domain='0x0000' bus='0x00' slot='0x03' function='0x0'/>
+              </interface>
+              <channel type='unix'>
+                <source mode='bind' path='/var/lib/libvirt/qemu/channels/78177d07-ac0e-4057-b1de-9ccd66cbc3d7.com.redhat.rhevm.vdsm'/>
+                <target type='virtio' name='com.redhat.rhevm.vdsm' state='connected'/>
+                <alias name='channel0'/>
+                <address type='virtio-serial' controller='0' bus='0' port='1'/>
+              </channel>
+              <input type='mouse' bus='ps2'>
+                <alias name='input0'/>
+              </input>
+            </devices>
+            <seclabel type='dynamic' model='dac' relabel='yes'>
+              <label>+107:+107</label>
+              <imagelabel>+107:+107</imagelabel>
+            </seclabel>
+          </domain>
+        </domstatus>
+    """
+    def parse_dom(self):
+        if self.dom is None:
+            return
+        else:
+            domstatus = {}
+            for child in self.dom:
+                if len(child) == 0:
+                    domstatus[child.tag] = child.text
+                else:
+                    domstatus[child.tag] = [c.items() for c in child]
+
+            return domstatus
+
+    @property
+    def vm_name(self):
```

Comment:
The same content as https://github.com/RedHatInsights/insights-core/pull/1632#discussion_r258813658

Reply:
Done (https://github.com/RedHatInsights/insights-core/pull/1703/commits/34f662efd44797ea0773d3b1606e7e855132612d)